server_start.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python power_systems_data_api_demonstrator/seed/seed.py && uvicorn power_systems_data_api_demonstrator.src.application:get_app --host 0.0.0.0 --port 10000
```

Comment:
What is this used by?

Reply:
It's going to be used by render to seed and then set up the app. I want to test if this works. If it works, then I can remove the `db.sqlite` from our repo (and new users will just have to run the seed script before starting).